README.md

```diff
@@ -1,2 +1,70 @@
-# cmo-metadb-label-generator
-CMO MetaDB messaging service for handling CMO label generation
+# CMO Label Generator 🔍
+
+CMO Label Generator is a distributed microservices system. It receives messages from LIMS when a request is marked delivered. This message is processed and a unique CMO label Id is generated. Any new message is then published to downstream subscribers.
+
+## Run
+
+### Custom properties
+
+Make an `application.properties` based on [application.properties.EXAMPLE](src/main/resources/application.properties.EXAMPLE).
+
+All properties are required with the exception of some NATS connection-specific properties. The following are only required if `nats.tls_channel` is set to `true`:
```

Comment:
I think we should be more explicit about the NATs [topic](https://github.com/mskcc/cmo-metadb-label-generator/blob/aa03efd8cbde61d99c444255d5d2efc9a740f4d6/src/main/resources/application.properties.EXAMPLE#L20) that has to be set to receive requests and the [one](https://github.com/mskcc/cmo-metadb-label-generator/blob/aa03efd8cbde61d99c444255d5d2efc9a740f4d6/src/main/resources/application.properties.EXAMPLE#L21) to get patient sample sets.